cypress/integration/active-spec.js

```diff
@@ -0,0 +1,31 @@
+import { Preview } from '../page-objects/preview-page'
+
+const preview = new Preview()
+
+describe( 'Check the Hover and Click Event', () => {
+
+	beforeEach( 'Open the English Page', () => {
+		cy.navigateToEnglishPage()
```

Comment:
Why is this spec using the English page instead of the test page, which appears to be a copy of the english page?

Reply:
we created the test page to add other cases (disambiguation, error, offline, etc...), this test uses the english page "just because" I believe, but @shailesh1028 can say more to this

Reply:
@jpita You have explained it right. The content of the test file is changed to get the other cases like disambiguation, error, etc. While the English file does not cover the above cases, hence we created the test file.